hibernate-envers/src/main/java/org/hibernate/envers/strategy/AuditStrategy.java

```diff
@@ -199,4 +202,32 @@ void addAssociationAtRevisionRestriction(
 			String alias1,
 			boolean inclusive,
 			MiddleComponentData... componentDatas);
+
+	/**
+	 * Method to add additional Columns to the aud-tables.
+	 *
+	 * @param anyMapping the class mapping
+	 * @param revMapping the mapping of the revInfo-relation
+	 * @param auditEntitiesConfiguration the configuration of the audit entities
+	 */
+	default void addAdditionalColumns(
+			Element anyMapping,
+			Element revMapping,
+			AuditEntitiesConfiguration auditEntitiesConfiguration) {
```

Comment:
I question if long-term a parameter object would be better served here.  In 6.0, I am eliminating the dom4j library and it will be replaced by the bootstrap process actually creating the Jaxb class instances themselves as a starting point to remove a lot of string-manipulation overhead.  By using a parameter object here, we would avoid tainting this method signature as we shift from 5.4 to 6.0.  I am not sure what I'd call such an object, so I need to think on it.